baseapp-content-feed/baseapp_content_feed/models.py

```diff
@@ -0,0 +1,35 @@
+import swapper
+
+from baseapp_core.graphql import RelayModel
+from django.conf import settings
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+
+class AbstractContentPost(
+    RelayModel
+):
+    author = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        verbose_name=_("author"),
+        related_name="content_posts",
+        on_delete=models.SET_NULL,
+        null=True,
+        blank=True,
+    )
+    content = models.TextField()
+
+
+    class Meta:
+        abstract = True
+
+
+class ContentPost(AbstractContentPost):
+    class Meta(AbstractContentPost.Meta):
+        abstract = False
+        swappable = swapper.swappable_setting("baseapp_content_feed", "ContentPost")
+
+
+SwappedContentPost = swapper.load_model(
```

Comment:
I think there is no need for this

Reply:
I import this elsewhere like
```python
from baseapp_content_feed.models import SwappedContentPost as ContentPost
```
thought it was nicer to use then calling `swapper.load_model(...)` directly.

Reply:
@nick-prat I think its better to use `swapper.load_model` just because here in the `models.py` file it needs `required=False, require_ready=False`, and its not needed in other places